reframe/frontend/executors/policies.py

```diff
@@ -80,7 +80,7 @@ def runcase(self, case):
             task.compile()
             task.compile_wait()
             task.run()
-            pollrate = PollRateFunction(0.2, 60)
+            sleeptime = itertools.cycle(range(1, 11))
```

Comment:
I think the equivalent to what we have now is the following:

```suggestion
            sleeptime = itertools.cycle(range(1, 10))
```

But we should make this a configuration parameter in any case I think.